src/features/DisplayControls/DrugChart/components/DrugChartView.jsx

```diff
@@ -87,15 +90,23 @@ export default function DrugChartWrapper(props) {
         endDateTimeInSeconds,
         visit
       );
-      setDrugChartData(response.data);
-      setIsShiftButtonsDisabled({
-        previous:
-          (isReadMode && response.data.length === 0) ||
-          response.data[0].startDate > startDateTimeInSeconds,
-        next:
-          startDateTimeInSeconds >= nextShiftMaxHour ||
-          endDateTimeInSeconds >= nextShiftMaxHour,
-      });
+      if (response?.error) {
+        if (response.error.response.status === 403) {
```

Comment:
can we move this status code to a src/constant file and use that from there?

Reply:
extracted to a constant